Remove material dependency

**Describe the bug**

The following error frequently occurs. Although it does not affect the hot reload functionality, it may cause discomfort to users. This is likely due to my project not using the Material dependency.

`Caused by: java.lang.ClassNotFoundException: androidx.compose.material.DividerKt`

**To Reproduce**

Attach a code snippet or test data if possible

**Expected behavior**

**Environment**
 - OS: [e.g. MacOS, Windows, Ubuntu] MacOS
 - Kotlin version: [e.g. `2.2.0`] 2.3.0
 - Compose Multiplatform version: [e.g. `1.8.2`] 1.10.0-rc02
 - Compose Hot Reload version: [e.g. `1.0.0-beta05`] 1.1.0-alpha03
 - IDE version (if bug is related to the IDE) [e.g. IntellijIDEA 2025.2, Android Studio 2025.1]  2025.3
   - Kotlin Multiplatform plugin version (if it is installed) [e.g. `0.9-253.14012-IJ_CE-0`]
 - Other relevant context

--- a/hot-reload-runtime-jvm/build.gradle.kts
+++ b/hot-reload-runtime-jvm/build.gradle.kts
@@ -65,6 +65,7 @@ kotlin.sourceSets.main.configure {
 }
 dependencies {
     compileOnly(deps.compose.desktop.common) {
+        exclude(group = "org.jetbrains.compose.material")
         version {
             strictly(deps.versions.composeMin.get())
         }

--- a/hot-reload-runtime-jvm/src/main/kotlinUI/org/jetbrains/compose/hotReloadUI/ErrorNotificationOverlayEffect.kt
+++ b/hot-reload-runtime-jvm/src/main/kotlinUI/org/jetbrains/compose/hotReloadUI/ErrorNotificationOverlayEffect.kt
@@ -33,7 +33,6 @@ import androidx.compose.foundation.layout.width
 import androidx.compose.foundation.layout.wrapContentWidth
 import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.foundation.text.BasicText
-import androidx.compose.material.Divider
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
@@ -56,8 +55,8 @@ import org.jetbrains.compose.devtools.api.ReloadAnimationSpec.statusColorFadeDur
 import org.jetbrains.compose.devtools.api.ReloadColors
 import org.jetbrains.compose.devtools.api.ReloadEffect
 import org.jetbrains.compose.devtools.api.ReloadState
+import org.jetbrains.compose.hotReloadUI.widgets.Divider
 import org.jetbrains.compose.hotReloadUI.widgets.TimeAgoText
-import org.jetbrains.compose.hotReloadUI.widgets.alignBox
 import org.jetbrains.compose.hot_reload.hot_reload_runtime_jvm.generated.resources.Res
 import org.jetbrains.compose.hot_reload.hot_reload_runtime_jvm.generated.resources.error
 import org.jetbrains.compose.resources.painterResource

--- a/hot-reload-runtime-jvm/src/main/kotlinUI/org/jetbrains/compose/hotReloadUI/widgets/Divider.kt
+++ b/hot-reload-runtime-jvm/src/main/kotlinUI/org/jetbrains/compose/hotReloadUI/widgets/Divider.kt
@@ -1,0 +1,36 @@
+/*
+ * Copyright 2024-2025 JetBrains s.r.o. and Compose Hot Reload contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
+
+package org.jetbrains.compose.hotReloadUI.widgets
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Composable
+internal fun Divider(
+    modifier: Modifier = Modifier,
+    color: Color = Color.White.copy(alpha = DividerAlpha),
+    thickness: Dp = 1.dp,
+    startIndent: Dp = 0.dp
+) {
+    val indent = if (startIndent.value != 0f) Modifier.padding(start = startIndent)
+    else Modifier
+
+    val targetThickness = if (thickness == Dp.Hairline)
+        (1f / LocalDensity.current.density).dp else thickness
+
+    Box(modifier.then(indent).fillMaxWidth().height(targetThickness).background(color = color))
+}
+
+private const val DividerAlpha = 0.12f

